tests/acceptance/test_performance_issues.py

```diff
@@ -29,18 +26,32 @@ def setUp(self):
         self.page = IssueDetailsPage(self.browser, self.client)
         self.dismiss_assistant()
 
-    @patch("django.utils.timezone.now")
-    def test_with_one_performance_issue(self, mock_now):
-        event = load_data("transaction")
-
-        data = json.loads(
-            self.load_fixture("events/performance_problems/n-plus-one-in-django-new-view.json")
+    def create_sample_event(self, start_timestamp):
+        event = json.loads(
+            self.load_fixture(
+                "events/performance_problems/n-plus-one-in-django-new-view.json"
+            ).decode("utf-8")
         )
```

Comment:
`json.loads` can take bytes directly -- and `.decode("utf-8")` is redundant for `.decode()` (though you should be able to remove the decode entirely)

Reply:
I saw that, but then the type annotation on `json.decode` insisted that it takes strings, even though it works on bytes! Should I update the type? https://github.com/getsentry/sentry/blob/df8fd27d8d98035869c8a491261cd2e34db9ee9c/src/sentry/utils/json.py#L109-L114